README.md

```diff
@@ -38,6 +38,7 @@ OpenRocket is an Open Source project licensed under the [GNU GPL](https://www.gn
 
 Contributing
 ------------
+[![SourceSpy Dashboard](https://sourcespy.com/shield.svg)](https://sourcespy.com/github/openrocketopenrocket/)
```

Comment:
@alexkarezin Can you move this to be after the Travis build status above and remove the Build Status header?

Reply:
@itshanney changed as requested